.github/workflows/ci-test-custom-script.yml

```
@@ -255,10 +255,14 @@ jobs:
           ./cypress/setup-test-ci.sh
 
       - uses: browser-actions/setup-chrome@latest
+        id: setup-chrome
         with:
-          chrome-version: stable
+          chrome-version: 129
+          install-chromedriver: true
+          install-dependencies: true
       - run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:70: Double quote to prevent globbing and word splitting

(shellcheck)